client/src/research/BubbleChart.css

```diff
@@ -7,9 +7,14 @@
   display:flex;
   flex-direction: row;
   align-items: center;
-  justify-content: space-evenly;
 }
 .Bubble-data {
+  display:flex;
```

Comment:
Like we talked about in person, elements like `table`, `tr` and `td` behave a bit differently if you assign `display: flex` to them.  Can you make sure those don't have that property set, and if you want to center things in the table cell, you can add a new `<div>` tag inside the cell and use flexbox for centering there.